build.gradle

```diff
@@ -47,10 +47,8 @@ apply plugin: 'project-report'
 apply plugin: 'build-dashboard'
 apply plugin: 'com.gradle.plugin-publish'
 
-tasks.withType(GroovyCompile) {
-    sourceCompatibility = '1.8'
-    targetCompatibility = '1.8'
-}
+sourceCompatibility = JavaVersion.VERSION_1_8
+targetCompatibility = JavaVersion.VERSION_1_8
```

Comment:
Note: You should be able to include these properties in the `java` extension closure on [line 77](https://github.com/dependency-check/dependency-check-gradle/pull/233/files#diff-49a96e7eea8a94af862798a45174e6ac43eb4f8b4bd40759b5da63ba31ec3ef7R77) (at least with recent major versions of Gradle). For example:

```gradle
java {
    sourceCompatibility = JavaVersion.VERSION_1_8
    targetCompatibility = JavaVersion.VERSION_1_8

    withJavadocJar()
    withSourcesJar()
}
```

References:
- https://docs.gradle.org/current/dsl/org.gradle.api.plugins.JavaPluginExtension.html#org.gradle.api.plugins.JavaPluginExtension:sourceCompatibility
- https://docs.gradle.org/current/dsl/org.gradle.api.plugins.JavaPluginExtension.html#org.gradle.api.plugins.JavaPluginExtension:targetCompatibility

Reply:
Thanks @elusiveeagle 

A thought before making the change:

Adding it `java` closure looks to have the same result on the `.module`, but I'm guessing it would mean re-introducing the `tasks.withType(GroovyCompile) {`.

Would it be better to leave it being set at `project` as both get there value from `project.sourceCompatibility`?

Reply:
Yeah thinking it through that makes sense.